doofinder-for-woocommerce/includes/api/endpoints/class-endpoint-product.php

```diff
@@ -634,23 +634,25 @@ private static function get_variations($products_data){
         $products = array();
 
         foreach($products_data as $product){
-            $type       = $product["type"];
-            $attributes = $product["attributes"];
+            if (!isset($product["type"])) {
```

Comment:
It does not make sense for me. Maybe you wanted to include the opposite condition?

```suggestion
            if ( isset( $product["type"] ) ) {
```

Reply:
You are right!